data/store/mongo/mongo_datum.go

```diff
@@ -38,10 +31,6 @@ const (
 var ErrSelectorsInvalid = errors.New("selectors is invalid")
 
 func (d *DatumRepository) EnsureIndexes() error {
-	modifiedTime, err := time.Parse(time.RFC3339, ModifiedTimeIndexRaw)
```

Comment:
Can we still have a lower bound - e.g. only index data from the last two years?

Reply:
I didn't think about making it 2 years, that should work.